src/llmcompressor/pipelines/layer_sequential/pipeline.py

```diff
@@ -79,6 +80,9 @@ def run_pipeline(
                     output = layer(**inputs)
 
                     if layer_index < num_layers - 1:
-                        output = to_next_layer_kwargs(output, layers[layer_index + 1])
+                        next_layer = layers[layer_index + 1]
+                        output = to_next_layer_kwargs(output, next_layer)
```

Comment:
slightly unrelated but general question: considering one of the assumptions of this pipeline is the flow of inputs/outputs, what was the need for adding `to_next_layer_kwargs` initially?

Reply:
For the sake of standardization, we only pass activations as kwargs. `to_next_layer_kwargs` simply assigns kwarg names to the unnamed but ordered args from the previous layer